monkey/monkey_island/cc/repository/local_storage_file_repository.py

```diff
@@ -55,6 +56,10 @@ def open_file(self, unsafe_file_name: str) -> BinaryIO:
                 f'Error retrieving file "{unsafe_file_name}" from the repository: {err}'
             )
 
+    def delete_files_by_pattern(self, file_name_pattern: str):
+        for file_name in glob.iglob(f"{self._storage_directory}/{file_name_pattern}"):
+            self.delete_file(file_name)
```

Comment:
Here you're using a glob, but in `SingleFileRepository` you're using a regex. The interface needs to more explicitly define what "pattern" is and all implementers need to adhere to the same interface.